centinel/client.py

```diff
@@ -37,7 +37,7 @@ def get_input_file(self, experiment_name):
         return os.path.join(self.config['dirs']['data_dir'], input_file)
 
     def load_input_file(self, name):
-        input_file = self.get_input_file(name)
+        input_file = self.get_input_file("%s.txt" %(name))
```

Comment:
This is actually a bug fix. Currently server is distributing *.txt data files. For now centinel code is not working because of the file name mismatching. This change gives get_input_file() the correct file name (with .txt extension) instead of the wrong file name. And why should we check both extensions or any extensions? I guess the person who uploads experiments and data files to server should follow the instruction and only use one kind of file type, which is txt, since server is already doing so.